### ISSUE TITLE
NOTIFY_EMAIL

### ISSUE THREAD

**Issue by bashcode** [u007/eximmon#2]
Hello,

Is possible to send notify email with autenticated email?
Because emails are sending with root@hostname

Thanks you

**Comment by u007** [comment]
you want to support smtp with authentication for notification email?

**Comment by bashcode** [comment]
Correct!.

And it would also be ideal if the lock time could be specified.

Good Work!

**Comment by u007** [comment]
yes i wanted to add the lock time, havent got time to spend on it. wanted to add domain specific control for that too, if you have time, i will be glad to guide you where to add it :)
